pkg/scheduler/core/generic_scheduler.go

```diff
@@ -711,60 +706,9 @@ func (g *genericScheduler) prioritizeNodes(
 	}
 
 	var (
-		mu   = sync.Mutex{}
-		wg   = sync.WaitGroup{}
-		errs []error
+		mu = sync.Mutex{}
+		wg = sync.WaitGroup{}
```

Comment:
move those variables inside the extenders if statement (line 731), also please use ```var mu sync.Mutex``` and ```var wg sync.WaitGroup``` in place of the block declaration.

Reply:
I also prefer `var` as opposed to block declaration, that's just what was there. Updated